client/src/components/UserModal/UserModal.tsx

```diff
@@ -25,9 +25,8 @@ const ModalLabel: FC<ILabelProps> = ({ text }): ReactElement => {
    * @returns the hex code of the color to display
    */
   const getLabelColor = (colorText: string): string => {
-    const value_index: number = Object.keys(interestsColorEnum).indexOf(
-      colorText,
-    );
+    const value_index: number =
```

Comment:
Nit: let's try to keep everything `camelCase` :)

Reply:
omg when did i do this LOL. This was closer to school and I was still typing everything in C++ format LOL

Reply:
actually this doesn't even matter because I deleted this function in a different PR